tooling/cli/lib/deploy.js

```diff
@@ -208,7 +208,7 @@ async function gatherUsedTranslations() {
     const parserScript = path.join(__dirname, 'parse-translations.js');
     const exec = childProcess.exec;
 
-    const parseProcess = exec(`node ${parserScript}`);
+    const parseProcess = exec(`node -e "require('${parserScript}').updateTranslations()"`);
```

Comment:
@AlexONeillTI Apologies but think we got wires crossed. I wasn't necessarily saying to just revert it, just that there was no `updateTranslations` method exported from `parse-translations.js`. Think we would want this to be `mainFunction` now, since that calls the methods previously handled with the self-calling function?